src/app/pages/Home/Home.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useState } from 'react';
+import styles from './Home.module.css';
+import Header from '../../components/Header/Header';
+import HeaderSpacer from '../../components/HeaderSpacer/HeaderSpacer';
+import SearchBar from '../../components/SearchBar/SearchBar';
+import TagGroup from '../../components/TagGroup/TagGroup';
+import { LANGUAGES } from '../../components/lib/languageMap';
+import CodeCard from '../../components/CodeCard/CodeCard';
+import Navigation from '../../components/Navigation/Navigation';
+
+export default function Home(): JSX.Element {
+  const [selectedLanguage, setSelectedLanguage] = useState<string>('html');
+  const [searchValue, setSearchValue] = useState<string>('');
+
+  const languageList = Object.values(LANGUAGES);
+  const tagLanguageList = languageList.map((language) => {
+    return {
+      children: language,
+      onClick: () => setSelectedLanguage(language),
+      active: selectedLanguage === language,
+    };
+  });
```

Comment:
The mapping of the languages could be done inside the ´TagGroup´ component and only the control part should be outside of it

Reply:
@phlgr